float32 obj_x
float32 obj_y
float32 obj_th
---
float32[] pbp

